examples/iotCore.ts

```diff
@@ -3,6 +3,6 @@ import { Handler } from "../src";
 export const handler: Handler.IotCore = (event) => {
     return {
         statusCode: 200,
-        body: event.messages[0]?.details.mqtt_topic.toString() || '',
+        body: event.messages[0]?.details.mqtt_topic.toString() || "",
```

Comment:
I think changing quotes style should be done in separate PR if really needed.

Reply:
This is rather not a change in style, it's a harmonization since they were not consistent. Since this is not a breaking change I would prefer not extracting it to a separate request :)